test/WebJobs.Script.Tests/Rpc/FunctionDispatcherTests.cs

```diff
@@ -221,12 +221,32 @@ public async void FunctionDispatcher_Restart_ErroredChannels_ExcceedsLimit()
                 foreach (var channel in functionDispatcher.JobHostLanguageWorkerChannelManager.GetChannels())
                 {
                     TestLanguageWorkerChannel testWorkerChannel = channel as TestLanguageWorkerChannel;
-                    testWorkerChannel.RaiseWorkerError();
+                    testWorkerChannel.RaiseWorkerErrorWithException();
                 }
             }
             Assert.Equal(0, functionDispatcher.JobHostLanguageWorkerChannelManager.GetChannels().Count());
         }
 
+        [Fact]
+        public async void FunctionDispatcher_Restart_ErroredChannels_NotAffectedByLimitWhenNoException()
+        {
+            int expectedProcessCount = 2;
+            FunctionDispatcher functionDispatcher = (FunctionDispatcher)GetTestFunctionDispatcher(expectedProcessCount.ToString());
```

Comment:
- Do not have to use multiple process. Remove expectedProcessCount , process count defaults to 1.
- Update test to throw combination of exit codes: 200 and non 200, this would mimic what would happen when a function app runs in azure.

Reply:
We still need to pass something to `WaitForJobhostWorkerChannelsToStartup`, even if we make it `1`, right?